appmap/django.py

```diff
@@ -211,19 +212,22 @@ def normalize_path_info(path_info, resolved):
 
 class Middleware(AppmapMiddleware):
     def __init__(self, get_response):
+        super().__init__()
         self.get_response = get_response
         self.recorder = Recorder()
-        self.record_url = "/_appmap/record"
 
     def __call__(self, request):
-        if not Env.current.enabled:
+        if not (
+            DetectEnabled.should_enable("remote")
+            or DetectEnabled.should_enable("requests")
```

Comment:
You could DRY up the code in the individual frameworks by adding a method to AppmapMiddleware like
```python
    @classmethod
    def should_record(cls):
        return DetectEnabled.should_enable("remote")
        or DetectEnabled.should_enable("requests")
```
then saying
```python
    if not self.should_record():
```
in the subclasses.